src/parser/parseSVGDocument.ts

```diff
@@ -62,8 +70,11 @@ export function parseSVGDocument(
     ); // http://www.w3.org/TR/SVG/struct.html#DefsElement
   });
   if (!elements || (elements && !elements.length)) {
-    callback([], {}, [], descendants);
-    return;
+    return {
+      ...emptyResponse,
```

Comment:
even spreading isn't safe

Reply:
i ll use clone

Reply:
createEmptyRespose sounds straight forward.
Cloning means someine needs to remember that